plugins/woocommerce/includes/abstracts/abstract-wc-order.php

```diff
@@ -19,6 +19,7 @@
 use Automattic\WooCommerce\Utilities\ArrayUtil;
 use Automattic\WooCommerce\Utilities\NumberUtil;
 use Automattic\WooCommerce\Utilities\OrderUtil;
+use Automattic\WooCommerce\Enums\ProductTaxStatus;
```

Comment:
My understanding is that we *generally* arrange `use` statements alphabetically. 

To that end, what do you think of moving this up a few lines (so it comes right after `Enums\OrderStatus`? That will also 'group' it nicely with the other enum classes we are importing.

It would be best to apply this same principle to all files we're touching.